File: jmanaloto9.c
//Joseph Manaloto
//Lab #9

#include <stdio.h>
#include <stdlib.h>
#include <time.h>

int main (void)
{
	srand(time(NULL));
	int i = 0;
	int guess;

	//this will guess between 1 to 20
	int answer = (rand() %20) + 1;

	//If number is out of range.
	
	//Actual guessing format.
	while(guess != answer)
	{
		printf("Guess the number between 1-20. ");
		scanf("%d", &guess);

		if(guess < 1 || guess > 20)
		{
			printf("Invalid range. Pick a number between 1-20. \n");
		}

		else if(guess > answer)
		{
			printf("Guess is incorrect, your guess is greater than the expected number. Your number is: %d ", guess);
		}
		else if(guess < answer)
		{
			printf("Guess is incorrect, your guess is less than the expected number. Your number is: %d ", guess);
		}
		else if(guess == answer)
		{
			for(i = 0; i < answer; i++)
			{
				printf("Guess is CORRECT!!!!!!!! Here's your w. \n");
			}
		}
	}

	return EXIT_SUCCESS;
}

File: jmanaloto9.java
//Joseph Manaloto
//csc321
//lab#5
//

import java.util.Scanner;
import java.util.Random;

public class jmanaloto9
{
	public static void main(String[] args)
	{
		Scanner input = new Scanner(System.in);

		int i = 0;
		int guess = 0;
		Random rand = new Random();

		int answer = rand.nextInt(20) + 1;

		while (guess != answer)
		{
			System.out.println("Pick a number between 1-20. ");
		if (input.hasNextInt())
		{
			guess = input.nextInt();
		
		if (guess < 1 || guess > 20)
		{
			System.out.println("Invalid range. Pick a number between 1-20. ");
		}
		else if (guess > answer)
		{
			System.out.println("Guess is incorrect, the number is greater than the answer: " + guess); 
		}
		else if (guess < answer)
		{
			System.out.println("Guess is incorrect, the number is lesser than the answer. " + guess);
		}
		else if ( guess == answer)
		{
			for (i=0; i< answer;i++)
			{
				System.out.println("Guess it correct!");
			}
		}
		}
		}


	}
}
